modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/rollback-to-openshift-sdn.adoc
+
+[id="nw-ovn-kubernetes-rollback_{context}"]
+= Rolling back the network provider to OpenShift SDN
+
+As a cluster administrator, you can rollback the Pod network provider for your cluster to the OpenShift SDN Pod network provider.
+During the rollback, you must reboot every node in your cluster.
```

Comment:
Should this information be in the assembly to give earlier warning?

Reply:
@mburke5678, the reboot?

Reply:
Including as

> IMPORTANT: During the migration, you must reboot every node in your cluster.

Reply:
But I actually have a warning already about workload interruptions, so maybe not.